databases.go

```diff
@@ -85,11 +87,18 @@ type DatabaseEngine struct {
 
 // DatabaseMaintenanceWindow stores information about a MySQL cluster's maintenance window
 type DatabaseMaintenanceWindow struct {
-	DayOfWeek   DatabaseDayOfWeek            `json:"day_of_week"`
-	Duration    int                          `json:"duration"`
-	Frequency   DatabaseMaintenanceFrequency `json:"frequency"`
-	HourOfDay   int                          `json:"hour_of_day"`
-	WeekOfMonth *int                         `json:"week_of_month"`
+	DayOfWeek DatabaseDayOfWeek            `json:"day_of_week"`
+	Duration  int                          `json:"duration"`
+	Frequency DatabaseMaintenanceFrequency `json:"frequency"`
+	HourOfDay int                          `json:"hour_of_day"`
+
+	Pending []struct {
+		Deadline    *time.Time `json:"deadline"`
+		Description string     `json:"description"`
+		PlannedFor  *time.Time `json:"planned_for"`
+	} `json:"pending,omitempty"`
```

Comment:
Any chance pending could be separated out to a separate struct? This would make things a bit easier in downstream implementations

Reply:
Yeah no problem. I will make that change rn.

Reply:
Fixed